File: LeetCode/Basic/1598/main.py
from typing import List


class Solution:
    def minOperations(self, logs: List[str]) -> int:
        cnt = 0
        for log in logs:
            if log == "../":
                if cnt != 0:
                    cnt -= 1
            elif log == "./":
                cnt -= 0
            else:
                cnt += 1

        return cnt

File: LeetCode/Basic/1598/main.go
package main

func minOperations(logs []string) int {
	var cnt int
	for _, log := range logs {
		switch log {
		case "../":
			if cnt != 0 {
				cnt -= 1
			}
		case "./":
			cnt += 0
		default:
			cnt += 1
		}
	}
	return cnt
}
